src/views/ResetPassword.vue

```diff
@@ -61,8 +69,17 @@ export default {
     };
   },
   methods: {
-    handleSubmit() {
-      console.log(this.model);
+    async onSubmit() {
+      this.model.email = this.$route.query.email;
```

Comment:
I suggest you to init the model when this page is rendered.
Also, you can use get method from lodash.
`this.model.email = get(this.$route, 'query.email', '')`